### PR TITLE
[Excel] (Custom Functions) Update TOC and article names

### PR DESCRIPTION
**Pull request by alison-mk**
This PR includes:
- Reorganization of the Custom Functions TOC, to
  - deprioritize the manual JSON metadata process
  - emphasize the autogenerate JSON metadata process
  - create a more logical structure, from basic to more complex concepts 
  - group tutorials/quick starts together 
- Adjustment of the manual JSON metadata article title, to clarify the distinction between autogenerate/manual JSON metadata processes, and then carrying this new title throughout the docset 
- Minor adjustment of a few other titles and link updates
- Minor typo/grammar updates

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[763d0dc](https://github.com/OfficeDev/office-js-docs-pr/commits/763d0dc838d6362a4093b88bfcfc8dcf9fc13f73)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-json-autogeneration.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-toc-refactor/docs/excel/custom-functions-json-autogeneration.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-json-autogeneration?branch=pr-en-us-2312) |
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-toc-refactor/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-json?branch=pr-en-us-2312) |
[docs/excel/custom-functions-localize.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-toc-refactor/docs/excel/custom-functions-localize.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-localize?branch=pr-en-us-2312) |
[docs/excel/custom-functions-naming.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-toc-refactor/docs/excel/custom-functions-naming.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-naming?branch=pr-en-us-2312) |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-toc-refactor/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-overview?branch=pr-en-us-2312) |
[docs/excel/custom-functions-parameter-options.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-toc-refactor/docs/excel/custom-functions-parameter-options.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-parameter-options?branch=pr-en-us-2312) |
[docs/excel/custom-functions-troubleshooting.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-toc-refactor/docs/excel/custom-functions-troubleshooting.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-troubleshooting?branch=pr-en-us-2312) |
[docs/excel/custom-functions-volatile.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-toc-refactor/docs/excel/custom-functions-volatile.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-volatile?branch=pr-en-us-2312) |
[docs/excel/custom-functions-web-reqs.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-toc-refactor/docs/excel/custom-functions-web-reqs.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-web-reqs?branch=pr-en-us-2312) |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-toc-refactor/docs/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-json?branch=pr-en-us-2312) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C11%5C7%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202011070113048545-2312%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=vHg%2F5lh3PYQlKSBjYUMiP62x0xb1fnUfDcv8t2Gbcqs%3D&st=2020-11-07T01%3A08%3A36Z&se=2020-12-08T01%3A13%3A36Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[3b8e855](https://github.com/OfficeDev/office-js-docs-pr/commits/3b8e8553be5e7698dbe98e00a9a0631280c52a74)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-json-autogeneration.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-toc-refactor/docs/excel/custom-functions-json-autogeneration.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-json-autogeneration?branch=pr-en-us-2312) |
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-toc-refactor/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-json?branch=pr-en-us-2312) |
[docs/excel/custom-functions-localize.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-toc-refactor/docs/excel/custom-functions-localize.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-localize?branch=pr-en-us-2312) |
[docs/excel/custom-functions-naming.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-toc-refactor/docs/excel/custom-functions-naming.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-naming?branch=pr-en-us-2312) |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-toc-refactor/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-overview?branch=pr-en-us-2312) |
[docs/excel/custom-functions-parameter-options.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-toc-refactor/docs/excel/custom-functions-parameter-options.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-parameter-options?branch=pr-en-us-2312) |
[docs/excel/custom-functions-shared-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-toc-refactor/docs/excel/custom-functions-shared-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-shared-overview?branch=pr-en-us-2312) |
[docs/excel/custom-functions-troubleshooting.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-toc-refactor/docs/excel/custom-functions-troubleshooting.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-troubleshooting?branch=pr-en-us-2312) |
[docs/excel/custom-functions-volatile.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-toc-refactor/docs/excel/custom-functions-volatile.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-volatile?branch=pr-en-us-2312) |
[docs/excel/custom-functions-web-reqs.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-toc-refactor/docs/excel/custom-functions-web-reqs.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-web-reqs?branch=pr-en-us-2312) |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-toc-refactor/docs/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-json?branch=pr-en-us-2312) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C11%5C7%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202011070116006847-2312%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=M1xMLGz4QrVx%2BY00x65UaGFmqXAVPUPEbI8OfDMXyj0%3D&st=2020-11-07T01%3A11%3A41Z&se=2020-12-08T01%3A16%3A41Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/toc.yml [hunk 1]
```diff
@@ -674,51 +674,50 @@
...
       displayName: Excel, Custom Functions
-    - name: Call Excel JS APIs from your custom function
-      href: excel/call-excel-apis-from-custom-function.md
+    - name: Parameter options
+      href: excel/custom-functions-parameter-options.md
       displayName: Excel, Custom Functions
     - name: Error handling
       href: excel/custom-functions-errors.md
       displayName: Excel, Custom Functions
+    - name: Autogenerate JSON metadata for custom functions
```
Comment (davidchesnut) [inline docs/toc.yml:700]:
```suggestion
    - name: Autogenerate JSON metadata
```
When were under the "custom functions" node I think we can leave that part out.

File: docs/toc.yml [hunk 2]
```diff
@@ -674,51 +674,50 @@
...
+      href: excel/custom-functions-json-autogeneration.md
+      displayName: Excel, Custom Functions
+    - name: Call Excel JS APIs from your custom function
```
Comment (davidchesnut) [inline docs/toc.yml:703]:
```suggestion
    - name: Call Excel JS APIs
```
```suggestion
    - name: Call Excel JS APIs from your custom function
```

File: docs/toc.yml [hunk 3]
```diff
@@ -674,51 +674,50 @@
...
+      href: excel/call-excel-apis-from-custom-function.md
+      displayName: Excel, Custom Functions
     - name: Identify an equivalent XLL
       href: excel/make-custom-functions-compatible-with-xll-udf.md
       displayName: Excel, Custom Functions
     - name: Localization
       href: excel/custom-functions-localize.md
       displayName: Excel, Custom Functions
-    - name: Metadata
+    - name: Manually create JSON metadata for custom functions
```
Comment (davidchesnut) [inline docs/toc.yml:712]:
```suggestion
    - name: Manually create JSON metadata
```
```suggestion
    - name: Manually create JSON metadata for custom functions
```

**Review by davidchesnut — CHANGES_REQUESTED** [review]
This looks good! Only suggestion is that we don't need to repeat term "custom functions" in the TOC since those articles are already under the "custom functions" node (so it is implied.)

**Comment by o365devx** [comment]
Docs Build status updates of commit _[b516652](https://github.com/OfficeDev/office-js-docs-pr/commits/b5166523e0ed2f87619fc3a55cbe5f7063cf7d9f)_: 

:clock10: **Pending**: waiting for processors ([11](https://ops.microsoft.com/#/statistic) builds ahead of you) 

 :warning: **Docs Build is busy**, currently there are 11 builds ahead of this one, for more information you can view the Build queue graph on the [Docs Portal](https://ops.microsoft.com/#/statistic).

**Comment by o365devx** [comment]
Docs Build status updates of commit _[b516652](https://github.com/OfficeDev/office-js-docs-pr/commits/b5166523e0ed2f87619fc3a55cbe5f7063cf7d9f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-json-autogeneration.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-toc-refactor/docs/excel/custom-functions-json-autogeneration.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-json-autogeneration?branch=pr-en-us-2312) |
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-toc-refactor/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-json?branch=pr-en-us-2312) |
[docs/excel/custom-functions-localize.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-toc-refactor/docs/excel/custom-functions-localize.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-localize?branch=pr-en-us-2312) |
[docs/excel/custom-functions-naming.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-toc-refactor/docs/excel/custom-functions-naming.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-naming?branch=pr-en-us-2312) |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-toc-refactor/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-overview?branch=pr-en-us-2312) |
[docs/excel/custom-functions-parameter-options.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-toc-refactor/docs/excel/custom-functions-parameter-options.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-parameter-options?branch=pr-en-us-2312) |
[docs/excel/custom-functions-shared-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-toc-refactor/docs/excel/custom-functions-shared-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-shared-overview?branch=pr-en-us-2312) |
[docs/excel/custom-functions-troubleshooting.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-toc-refactor/docs/excel/custom-functions-troubleshooting.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-troubleshooting?branch=pr-en-us-2312) |
[docs/excel/custom-functions-volatile.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-toc-refactor/docs/excel/custom-functions-volatile.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-volatile?branch=pr-en-us-2312) |
[docs/excel/custom-functions-web-reqs.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-toc-refactor/docs/excel/custom-functions-web-reqs.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-web-reqs?branch=pr-en-us-2312) |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-toc-refactor/docs/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-json?branch=pr-en-us-2312) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C11%5C11%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202011110007430038-2312%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=0SLAEYLKi2t%2FQqlX0hF%2FafiaRqXDzDo3SUhThaAgHfU%3D&st=2020-11-11T00%3A03%3A22Z&se=2020-12-12T00%3A08%3A22Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/excel/custom-functions-shared-overview.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ title: Run your add-in code in a shared JavaScript runtime
 localization_priority: Priority
 ---
 
-# Overview: Run your add-in code in a shared JavaScript runtimes
+# Overview: Run your add-in code in a shared JavaScript runtime
```
Comment (mscharlock) [inline docs/excel/custom-functions-shared-overview.md:8]: Man, sorry about this one! smh

**Review by mscharlock — APPROVED** [review]
Looks good to me :)

**Review by davidchesnut — APPROVED** [review]
Looks great!

**Pull request merged**